setup.py

```diff
@@ -42,7 +63,7 @@
     include_package_data=True,
 
     # Allows `setup.py test` to work correctly with pytest
-    setup_requires=[] + pytest_runner,
+    setup_requires=['numpy>=1.20.2'] + pytest_runner,
```

Comment:
That's a fairly new version of NumPy--are you leveraging some bleeding edge features for the analysis work?

Reply:
As a general rule, be as lenient about your required versions as possible. It can be very difficult for a package manager to resolve specific version requirements, especially on slightly older installations with very new package versions.

Reply:
Thanks for this. I wrongly assumed the NumPy version required should match the one used in the environment the tool is developed. In this case, `numpy>=1.16` will do.